src/tensors/cpu/expression_graph_packable.h

```diff
@@ -1,10 +1,57 @@
 #pragma once
 
 #include "graph/expression_graph.h"
-#include "packed_gemm.h"
+#include "fbgemm/packed_gemm.h"
+#include "tensors/cpu/integer_common.h"
+
+namespace { //Temporary annonymous transposition, until we figure out how to access the proper one
+inline void transpose4x4_SSE(const float* A,
+                             float* B,
+                             const int lda,
+                             const int ldb) {
+  __m128 row1 = _mm_load_ps(&A[0 * lda]);
+  __m128 row2 = _mm_load_ps(&A[1 * lda]);
+  __m128 row3 = _mm_load_ps(&A[2 * lda]);
+  __m128 row4 = _mm_load_ps(&A[3 * lda]);
+  _MM_TRANSPOSE4_PS(row1, row2, row3, row4);
+  _mm_store_ps(&B[0 * ldb], row1);
+  _mm_store_ps(&B[1 * ldb], row2);
+  _mm_store_ps(&B[2 * ldb], row3);
+  _mm_store_ps(&B[3 * ldb], row4);
+}
+
+// from
+// https://stackoverflow.com/questions/16737298/what-is-the-fastest-way-to-transpose-a-matrix-in-c
+#define ROUND_UP(x, s) (((x) + ((s)-1)) & -(s))
+
+void Transpose10(marian::Tensor out, const marian::Tensor in) {
```

Comment:
I need to access a transpose operation in the `expression_graph_packable.h`. As far as I could tell, the normal `tranpose` function, requires a `graph->forward()` and I couldn't figure out how to do this just for one tensor and one operation. Is that possible?

Reply:
Since this is a copy, why would the copy work and the original wouldn't? Or what am I missing?

Reply:
Because the original is defined in `tensors/cpu/tensor_operators.cpp` and can't be included in this file. The alternative is to move the transposes to a separate header, but that change was even more invasive. I thought you'd know a better way of doing it.

Reply:
How about doing a slow transpose without any vectorized code? By using two for loops. This is always done offline and it might still be within a few seconds even with a plain transpose. @XapaJlaMnu @emjotde